monai/handlers/mlflow_handler.py

```diff
@@ -88,6 +101,11 @@ def __init__(
         global_epoch_transform: Callable = lambda x: x,
         state_attributes: Optional[Sequence[str]] = None,
         tag_name: str = DEFAULT_TAG,
+        experiment_name: str = "default_experiment",
+        run_name: Optional[str] = None,
+        experiment_param: Optional[Dict] = None,
+        artifacts: Optional[Sequence[Path]] = None,
+        optimizer_param_names: Tuple[str, ...] = ("lr",),
```

Comment:
Please use `ensure_tuple` to simplify the arg value, for example:
```py
optimizer_param_names: Union[str, Sequence[str]] = "lr"
```
Then:
```py
self.optimizer_param_names = ensure_tuple(optimizer_param_names)
```

Thanks.